src/amplifyClient.ts

```diff
@@ -216,6 +216,23 @@ export const queries = {
       }
     }
   `,
+  getVotingRepInstallations: /* GraphQL */ `
+    query GetVotingRepInstallations(
+      $votingRepHash: String!
+      $colonyAddress: ID!
+    ) {
+      listColonyExtensions(
+        filter: {
+          hash: { eq: $votingRepHash }
+          colonyId: { eq: $colonyAddress }
+        }
+      ) {
+        items {
+          id
+        }
+      }
+    }
+  `,
```

Comment:
That might break relatively soon. By default, it'll only get the first 100 of all the extensions ever installed and filter those for the right hash and colony address.
Have a look at `getExtensionByColonyAndHash` query, despite its slightly imprecise name it should return all installations for a given colony and hash using indexes on these fields instead of filtering